packages/core/test/staticClient/headers.test.ts

```diff
@@ -258,4 +259,30 @@ describe("Base Client headers", () => {
       expect(nock.isDone()).to.be.true;
     });
   });
+
+  describe("getObjectFromResponse", () => {
+
+    it("should return a map of headers", () => {
```

Comment:
```suggestion
    it("returns a map of headers", () => {
```
As per best practices style guide.